src/apps/dialogs/SavingThrowRollDialog.svelte

```diff
@@ -74,9 +80,10 @@
     }
 
     let visibilityMode = options.visibilityMode ?? game.settings.get("core", "rollMode");
-
     let saveType = options.saveType ?? "standard";
+    let { minRoll } = options.saveMinRoll ?? $actor.system.abilities[abilityKey].save.minRoll;
```

Comment:
Default to 1.
Can't extract `minRoll` from `options.minRoll`.
The ability key can be null in the case of death save.
`options.minRoll` is sufficient. There's no way to make checks and saves in the same dialog.

```ts
let minRoll = options.minRoll ?? $actor.system.abilities[abilityKey]?.save?.minRoll ?? 1;
```